src/test/java/nodejs/schemas/users/users.json

```diff
@@ -0,0 +1,7 @@
+{
+  "first_name": "#string",
+  "last_name": "#string",
+  "email": "#string",
+  "admin": "##boolean",
```

Comment:
if both fields are optional, the API may not return either

Reply:
I tried to apply something like 'oneOf' but couldn't find it with the Karate docs.
Finally, I just deleted one of them, 'admin', because in order to do an assertion I should know the expected result and not put data not defined yet.